pages/api/webhooks/customer-created.ts

```diff
@@ -23,8 +23,18 @@ const handler: Handler = async (request) => {
   }
 
   const klaviyoClient = Klaviyo(klaviyoToken);
-  klaviyoClient.send(klaviyoMetric, userEmail, context);
+  const klaviyoResponse = await klaviyoClient.send(
+    klaviyoMetric,
+    userEmail,
+    context
+  );
 
+  if (klaviyoResponse.status !== 200) {
+    return Response.InternalServerError({
+      success: false,
+      message: "Something went wrong.",
```

Comment:
Is message field available in the error response? https://developers.klaviyo.com/en/reference/api-overview#errors